rskj-core/src/main/java/co/rsk/peg/storage/BridgeStorageAccessorImpl.java

```diff
@@ -50,7 +50,7 @@ private <T> byte[] getSerializedData(T object, RepositorySerializer<T> serialize
     }
 
     @Override
-    public void saveToRepository(DataWord addressKey, byte[] serializedData) {
+    public void safeSaveToRepository(DataWord addressKey, byte[] serializedData) {
```

Comment:
Any reason this one is not typed in the interface?

Reply:
it is https://github.com/rsksmart/rskj/blob/fix/test/bridge_storage_provider_test/rskj-core/src/main/java/co/rsk/peg/storage/StorageAccessor.java#L11

Reply:
To use the first method, you have to pass a serialization method as an input.
The idea is that, eventually, each object knows how to serialize itself (this is already happening in the [PendingFederation](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/federation/PendingFederation.java#L172) class).

When that happens for all of the objects, we would no longer need to use the first method [but this one](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java#L360), since we will be able to have the serialized data.